lib/configuration_methods.rb

```diff
@@ -34,6 +34,22 @@ def configured_services
     end
     attr_writer :configured_services
 
+    def show_service?(service, user)
+      unless self["services.#{service}.enable"]
+        return false
+      end
+      # If service has an 'authorized' attr, return true only if it is true or equal to user username
+      if self["services.#{service}"].has_key?("authorized")
```

Comment:
I did try all the cases I could see and it seemed to work all throughout, but will change :)